reference/src/client/test/sa_key_provision_widevine.cpp

```diff
@@ -0,0 +1,348 @@
+/*
+ * Copyright 2020-2024 Comcast Cable Communications Management, LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
```

Comment:
See other comments in test files as there are many applicable here. We should use a similar test strategy as other SecAPI3 tests where the failure that is being tested is asserted. Looks like this applies to all tests in this PR. Use other secAPI3 client tests as a guide such as those for sa_key_import.